frontend/src/components/ConditionalFields.js

```diff
@@ -68,18 +63,6 @@ export default function ConditionalFields({
         const validationsAndCompletions = CONDITIONAL_FIELD_CONSTANTS[prompt.fieldType];
         const rules = validationsAndCompletions.transformValidationsIntoRules(prompt.validations);
 
-        const initialValue = (() => {
-          const current = initialValues.find((p) => p.promptId === prompt.promptId);
-          if (current) {
-            return current.response;
-          }
-
-          return [];
-        })();
-
-        const completions = validationsAndCompletions.confirmResponseComplete(prompt.validations);
-        const isComplete = completions.every((completion) => completion(initialValue));
-
```

Comment:
love to see all the red

Reply:
I don't have the lines in latest version did you pull?

Reply:
No idea what you mean. What I said was that I like to see that you've removed files.

Reply:
I guess what did you mean by this comment? I though you are saying I have unused lines of code. But I don't see these lines of code in latest version.

Reply:
sorry for the confusion, I was just saying I was glad to see you could remove all this code, no action required on your part